modules/bbp-electrophysiology/src/main/resources/schemas/bbp/electrophysiology/trace/v0.1.0.json

```diff
@@ -24,64 +24,81 @@
       "and":[
         {
           "node":"bbpentsh:EntityShape"
-        },{
+        },
+        {
           "property" : [ {
             "path": "nsg:projectName",
             "name": "Project name",
             "description": "Name of the project that the trace was recorded for",
             "datatype": "xsd:string",
             "maxCount": 1
-          }, {
+          },
+          {
             "path": "nsg:sweep",
```

Comment:
Is this sweep number the same as the one present in the trace generation. Shouldn't we keep the sweep number in one place ?